main/client/mainLayout.html

```diff
@@ -1,3 +1,8 @@
+<head>
+    <title>Refugee Tech</title>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="//f.fontdeck.com/s/css/438QB1C59GkIxOiyjaJyqETvpBo/refugeetech.com/20575.css" type="text/css" />
```

Comment:
The purpose was to fetch the fonts. This didn't work so it should have been removed before the pull request. Sorry about that